modules/friendlyshared.js

```diff
@@ -144,7 +150,7 @@ Twinkle.shared.callbacks = {
 			return;
 		}
 
-		Morebits.status.info( 'Info', 'Will add the shared IP address template to the top of the user\'s talk page.' );
+		Morebits.status.info( $.i18n._('tw-core-morebits-info'), $.i18n._('tw-sharedip-will-add') );
```

Comment:
Not `tw-core-morebits`. Just `morebits` will do. morebits.js is not Twinkle-specific.